### PR TITLE
Add requisites and installations to README

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,22 @@ It contains two parts:
 - A fluentbit daemonset: Small collector running on every node collecting containers and audit logs and for forwarding them to fluentd.
 - A fluentd deployment: Takes care of persisting the logs to any data sink
 
+## Requirements
+
+- You can install only one release of this chart per kubernetes cluster
+- By default any forwarded is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```
Comment (oponder) [inline README.md]: Is `forward` a thing that can be active? or should it be `forwarded logs are active`? or `any log forwarding is active` 

And what do you mean with `any` ? Should it be `all` ?

```suggestion
- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```

Comment (omegas27) [reply README.md]: I think it should be "no forward" as I had to disable all of them by default when I added Azure support

File: README.md [hunk 2]
```diff
@@ -8,9 +8,22 @@ It contains two parts:
 - A fluentbit daemonset: Small collector running on every node collecting containers and audit logs and for forwarding them to fluentd.
 - A fluentd deployment: Takes care of persisting the logs to any data sink
 
+## Requirements
+
+- You can install only one release of this chart per kubernetes cluster
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply README.md]: yeah now it is not possible but it is on the roadmap

File: README.md [hunk 3]
```diff
@@ -8,9 +8,22 @@ It contains two parts:
...
+- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```
Comment (oponder) [inline README.md]:
```suggestion
- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploying it in your cluster.
```

File: README.md [hunk 4]
```diff
@@ -8,9 +8,22 @@ It contains two parts:
...
+
+## Installation
+
+The logging forwarding app is built to be installed in AWS or Azure. For AWS support to send the logs to S3 and/or Cloudwatch. For Azure it uses Log Analytics:
```
Comment (oponder) [inline README.md]:
```suggestion
The logging forwarding app is built to be installed in AWS or Azure. On AWS the app can send logs to S3 and/or Cloudwatch. On Azure it uses Log Analytics.

Sample command for installing it on AWS with cloudwatch enabled:
```

**Pull request merged**